modules/administration-guide/partials/proc_backing-up-a-che-instance-by-using-prod-cli-and-a-CheBackupServerConfiguration-custom-object.adoc

```diff
@@ -0,0 +1,34 @@
+[id="backing-up-a-{prod-id-short}-instance-by-using-{prod-cli}-and-a-CheBackupServerConfiguration-custom-object_{context}"]
+= Backing up a {prod-short} instance by using {prod-cli} and a CheBackupServerConfiguration custom object
+
+You can back up a {prod-short} instance by entering the `name` of a `CheBackupServerConfiguration` custom object as an argument with the `{prod-cli} server:backup` command.
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Configured backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+* Created `CheBackupServerConfiguration` custom object.
+
+.Procedure
+
+. Run the following command on a command line:
++
+[subs="+quotes,+attributes"]
+----
+$ {prod-cli} server:backup \
+--backup-server-config-name=__<name_of_CheBackupServerConfiguration>__ <1>
```

Comment:
Why do we need a new line here?

Reply:
@mmorhun, :) so that I can use the `<1>` syntax (otherwise, it's all on one line).
I do hope readers don't take backslashes seriously.